assets/src/js/admin/onboarding-wizard/utils/index.js

```diff
@@ -9,6 +9,32 @@ export const getWindowData = ( value ) => {
 	return data[ value ];
 };
 
+/**
+ * Walks through input/button elements on the page and sets tabindex attributes for keyboard accessibility.
+ * Finishes by focusing browser on the first input in the step.
+ *
+ * @since 2.8.0
+ */
+export const setupTabIndex = () => {
+	const stepInputs = document.querySelectorAll( '.give-obw-step button, .give-obw-step input, .give-obw-step select' );
+	const navButtons = document.querySelectorAll( '.give-obw-step-navigation button' );
+	const formPreviewIframe = document.querySelector( '.give-obw-step iframe' );
+
+	stepInputs.forEach( ( input ) => {
+		input.setAttribute( 'tabindex', 1 );
+	} );
+
+	if ( formPreviewIframe !== null ) {
+		formPreviewIframe.setAttribute( 'tabindex', 2 );
```

Comment:
We might want to consider setting focus to the container first (even if the preview isn't loaded) and allow the user to tab to the "Continue" button and back to the container/preview. As it stands, the tab order is visually backwards: starting at the "Continue" button and then tabbing to the Preview.

Also, I think there are accessibility issues with setting a tab index greater than 1, so the _proper_ way is to semantically order the elements in the DOM and let the tab order sort itself.

Reply:
Good thinking, I played with the focus for this step both ways and couldn't decide which felt more useful - but I trust your instinct here. 

I've gone ahead and reworked the Wizard component so that step navigation appears below the steps in the DOM tree (but using flex-direction: column-reverse they are visually displayed along the top as originally intended). This fix also made the setupTabIndex utility unnecessary, so I've replaced it with a more simple utility (setStepFocus) to set the correct focus when a user advances to a new a step. These changes mean no elements in the Onboarding Wizard app have a specific tabindex attribute set, and that now, on step 4, the iframe is focused first, then users can tab out of it to the continue button if they wish.

Ref: https://github.com/impress-org/givewp/commit/3fc9450da98a90dbd720c3d3d672604e907ae0c9